.dotnet/src/Custom/Embeddings/EmbeddingTokenUsage.cs

```diff
@@ -1,10 +1,16 @@
 namespace OpenAI.Embeddings;
 
-[CodeGenModel("EmbeddingUsage")]
+[CodeGenModel("CreateEmbeddingResponseUsage")]
 public partial class EmbeddingTokenUsage
 {
     // CUSTOM: Renamed.
     /// <summary> The number of tokens used by the input prompts. </summary>
     [CodeGenMember("PromptTokens")]
-    public long InputTokens { get; }
+    public int InputTokens { get; }
+
+    // CUSTOM: Convenience member added for completeness.
+    /// <summary>
+    /// The number of tokens used by embedding outputs.
+    /// </summary>
+    public int OutputTokens => TotalTokens - InputTokens;
```

Comment:
I wonder if it would be advisable to not add custom properties to this class, mostly because I'm not sure how much value this approach to measuring usage is in the first place. That being said, I don't feel strongly.

Reply:
Can always add it later. I'll remove it.